src/app/authentication/login/login.component.html

```diff
@@ -1,110 +1,79 @@
-              </mat-error>
-            </mat-form-field>
-            <div *ngIf="loginFailed">
-              <span style="color:#f44336">
-                Invalid username or password.
-              </span>
-            </div>
-          </div>
-          <div fxLayout="column" style="margin-top:60px;">
-            <button (click)="login()" color="primary" mat-raised-button type="submit">LOGIN</button>
-            <span>&nbsp;</span>
-            <div fxLayout="row">
-              <span style="margin: 0 auto;">Don't have an account? <a routerLink="/authentication/register">Sign Up</a></span>
-            </div>
-          </div>
-        </form>
-      </div>
-    </mat-card>
-    <span fxFlex></span>
-  </div>
-</div>
-
-<!-- Mobile -->
-<div class="masthead-mobile" *ngIf="isMobile">
-  <div fxLayout="column" fxFlexFill style="margin-left:10px; margin-right:10px;">
-    <span style="margin-top: auto;">&nbsp;</span>
-    <div fxLayout="row" style="min-height:55px;">
-      <span fxFlex></span>
-      <i class="material-icons" style="font-size:60px; color: white;">account_balance</i>
-      <span fxFlex></span>
-    </div>
-    <div fxLayout="row" style="margin-top:10px;min-height:40px;">
-      <span fxFlex></span>
-      <div style="font-size: 40px; color: white;">
-        Sample App
+  <div fxLayout="row" fxLayoutAlign="center center" class="full-height">
+    <mat-card fxFlex="50" fxFlex.gt-md="40" fxLayout="column" fxLayoutGap="30px">
+      <div fxLayout="row" fxFlexAlign="center">
+        <h1>Sample App</h1>
       </div>
-      <span fxFlex></span>
-    </div>
-    <form [formGroup]="loginForm" (ngSubmit)="login()" novalidate style="margin-top: auto; margin-left:15px; margin-right: 15px;">
-      <div fxLayout="column">
-        <mat-form-field class="light-input">
-          <span matPrefix style="margin-right:10px">
-            <mat-icon style="color:white; font-size:26px;">person_outline</mat-icon>
-          </span>
+      <form [formGroup]="loginForm" (ngSubmit)="login()" fxLayout="column" fxLayoutGap="10px" novalidate>
+        <mat-form-field>
           <input matInput placeholder="Username" formControlName="username" required>
           <mat-error *ngIf="loginForm.controls['username'].hasError('required')">
             Username is required
           </mat-error>
         </mat-form-field>
-        <mat-form-field class="light-input" style="margin-top:15px">
-          <span matPrefix style="margin-right:10px">
-            <mat-icon style="color:white; font-size:26px;">lock_outline</mat-icon>
-          </span>
+        <mat-form-field>
           <input matInput placeholder="Password" formControlName="password" type="password" required>
           <mat-error *ngIf="loginForm.controls['password'].hasError('required')">
             Password is required
           </mat-error>
         </mat-form-field>
         <div *ngIf="loginFailed">
-          <span style="color:white">
+          <span style="color:#f44336">
```

Comment:
Seems like this one should be changed as well right?

Reply:
nice catch